sdk/containerregistry/Azure.Containers.ContainerRegistry/src/autorest.md

```diff
@@ -0,0 +1,9 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+input-file:
```

Comment:
I think this should be linking to https://github.com/Azure/azure-rest-api-specs/blob/master/specification/containerregistry/data-plane/readme.md

see the following example:
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/src/autorest.md

Reply:
Thanks, @christothes !  I'll work with @ShivangiReja on this.